flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -132,9 +134,11 @@ public HiveCatalog(String catalogName, @Nullable String defaultDatabase, @Nullab
 		this.hiveConf = hiveConf == null ? getHiveConf(null) : hiveConf;
 
 		LOG.info("Created HiveCatalog '{}'", catalogName);
+
+		HiveConf.setHiveSiteLocation(null);
```

Comment:
I don't think we should reset this. User passes us a HiveConf instance and we use it. We don't need to update either the class or the instance.